runestone/activecode/js/activecode.js

```diff
@@ -986,11 +986,14 @@ AudioTour.prototype.tour = function (divid, audio_type, bcount) {
         // str+="<audio id="+akey+" preload='auto'><source src='http://ice-web.cc.gatech.edu/ce21/audio/"+
         // akey+".mp3' type='audio/mpeg'><source src='http://ice-web.cc.gatech.edu/ce21/audio/"+akey+
         // ".ogg' type='audio/ogg'>Your browser does not support the audio tag</audio>";
+        
+        var dir = "http://media.interactivepython.org/studentcsp/audio/" 
```

Comment:
Barb,

We want to use ``eBookConfig.basecourse`` here instead of hardcoding the path to include studentcsp